### ISSUE TITLE
Similar to #520 slashes not stripped on double quotes in relation meta with an wysiwyg field

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#581]
```
Hi, I ran into an issue which is similar to #520. This time it has something to 
do with double quotes. Plugin version 0.9.7.13.

Follow these steps to reproduce this:

1. Create an relation meta with an wyswig field
2. When adding content to the field, create at least one link to an page (or 
just append an attribute with an value to an element of your choice)
3. Save
4. Now if you take a look at the attributes in the plain text mode you see that 
the quotes didn't got stripped.

For a quick fix i modified the file 
custom-content-type-manager/fields/relationmeta.php:469

Before returning the array:

$out = stripslashes_deep($out);
```

Original issue reported on code.google.com by `miron.og...@googlemail.com` on 26 Nov 2014 at 3:06

**Comment by GoogleCodeExporter** [comment]
```
Can you paste the exact changes you made?  And can you paste an example of what 
content you added to the field?
```

Original comment by `ever...@fireproofsocks.com` on 27 Nov 2014 at 5:19
- Changed state: **InReview**

**Comment by GoogleCodeExporter** [comment]
```
Hey, sure.

Here's the content which I include:

<h2>Title</h2>
Some text. More Text

 

<a title="Some title" href="http://www.google.com/">Link to Google</a>

I attached a patch file with the changes.
```

Original comment by `miron.og...@googlemail.com` on 28 Nov 2014 at 11:44

Attachments:
- [0001-temporary-fix-for-the-slashes.patch](https://storage.googleapis.com/google-code-attachments/wordpress-custom-content-type-manager/issue-581/comment-2/0001-temporary-fix-for-the-slashes.patch)

**Comment by GoogleCodeExporter** [comment]
```
I attached here the updated file which has the fix for this issue.
```

Original comment by `daniel.e...@gmail.com` on 19 Dec 2014 at 12:10

Attachments:
- [relationmeta.php](https://storage.googleapis.com/google-code-attachments/wordpress-custom-content-type-manager/issue-581/comment-3/relationmeta.php)

**Comment by GoogleCodeExporter** [comment]
```
Released in 0.9.8
```

Original comment by `ever...@fireproofsocks.com` on 6 May 2015 at 11:52
- Changed state: **Fixed**
